cypress/integration/google.spec.ts

```diff
@@ -3,4 +3,4 @@ describe('This is my first cypress test', () => {
         cy.visit('https://www.google.com/');
         cy.title().should('eq', 'Google');
     });
-});
\ No newline at end of file
+});
```

Comment:
In the local file is the empty line at the end

Reply:
you're right